gokart/task.py

```diff
@@ -49,7 +49,6 @@ class TaskOnKart(luigi.Task):
     fix_random_seed_value = luigi.IntParameter(default=None, description='Fix random seed method value.', significant=False)
 
     def __init__(self, *args, **kwargs):
-        self._add_configuration(kwargs, self.get_task_family())
```

Comment:
I think just introducing TaskOnKart.xxx (following line) will demand #6, and removed this line. Please let me know if this will break any other situations.

Reply:
I agrre with your suggestion. LGTM!